src/components/pages/PendingApplications/MenteeModal.js

```diff
@@ -0,0 +1,123 @@
+import { List, Tag } from 'antd';
+import '../../../styles/styles.css';
+import './PendingApplication.css';
+
+const MenteeModal = ({ applicant }) => {
+  return (
+    <>
+      <div className="profile-intro">
+        <div className="image-container">
+          <img
+            src="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png"
+            alt="I can do this if I work hard enough and practice my coding skills"
+          />
+        </div>
+        <div className="profile-intro-description">
+          <h2>
+            {applicant.first_name} {applicant.last_name}
+          </h2>
+          <div className="preferences">
+            <div>
+              <p>Interested in: </p>
+            </div>
+            <div className="mentee-tag">
+              <Tag color={'purple'}>{applicant.tech_stack}</Tag>
+            </div>
+          </div>
+        </div>
+      </div>
+      <List size="small">
+        <List.Item className="list-item">
+          <div className="list-item-column">
+            <p>Role</p>
+          </div>
+          <div className="list-item-column">
+            <p>Mentee</p>
+          </div>
+        </List.Item>
+        <List.Item className="list-item">
+          <div className="list-item-column">
+            <p>Email:</p>
+          </div>
+          <div className="list-item-column">
+            <p>{applicant.email}</p>
+          </div>
+        </List.Item>
+        <List.Item className="list-item">
+          <div className="list-item-column">
+            <p>Location:</p>
+          </div>
+          <div className="list-item-column">
+            <p>
+              {applicant.state} {applicant.country}
+            </p>
+          </div>
+        </List.Item>
+        <List.Item className="list-item">
+          <p className="list-item-column">Membership Criteria:</p>
+          <div className="list-item-column">
+            {applicant.formerly_incarcerated === true ||
+            applicant.low_income === true ||
+            applicant.underrepresented_group === true ? (
```

Comment:
Overuse or nested ternary conditionals can be difficult to read / understand. Perhaps using a mapping object to display certain text would be a good alternative? 
```
const criteria = [
  {criteria: applicant.formerly_incarcerated, text: "Formerly Incarcerated"},
  {criteria: applicant.low_income, text: "Low Income"},
  {criteria: applicant.underrepresented_group, text: "Belongs to underrepresented group"},
]

criteria.filter(c => c.criteria).forEach(c => <li>{c.text}</li>)
```
My argument is that if the only thing that's changing is the text in an `<li>` tag, then looping through a list of criteria would be the easiest way to be able to potentially add more items in the future (just add to or adjust the array / object);